common/context.go

```diff
@@ -70,8 +70,11 @@ func GetProcessID(ctx context.Context) int {
 func contextWithDoneChan(ctx context.Context, done chan struct{}) context.Context {
```

Comment:
One thing I forgot to mention (and I can't comment on directly): should we remove the "The context will leak if the done channel is never closed" sentence from the comment, since we'll now cancel it if the the main context is done?

Reply:
Right 🤦 I'll remove it and directly push it to `main`.

Reply:
See: https://github.com/grafana/xk6-browser/commit/917b83ef14b3e2075f92c092163c47e359883a6a